### ISSUE TITLE
[Bug] Event Token Drawbox, Arcarum Sandbox and Raid Navigation Errors

### ISSUE THREAD

**Issue by ChikyuTenshi** [christianciu/gbf-automation-updated#14]
Hey Chris, Bot seems to be getting stuck after a team wipe because it can't handle the "Battle in progress" after backing out without retreating

![Screenshot 2024-05-05 002905](https://github.com/christianciu/gbf-automation-updated/assets/20892295/fe3ba98c-1ba9-4599-89bf-d1e8b567c81c)

And here's a [Pastebin ](https://pastebin.com/bAcBPtCU)with the debug logs

Solution: Make it stay in raid after wipe and send backup requests until clear

**Comment by ChikyuTenshi** [comment]
@christianciu  Alright things seems to be bugging out for me way too much during navigation

Here it can't see the Arcarum_Banner, Scrolling all the way to the bottom and then getting stuck.

https://github.com/christianciu/gbf-automation-updated/assets/20892295/5939ba43-1820-4500-b683-4232003dc700



And here trying to run Impossible Tiamat Omega and getting stuck in a loop, Also it will join raids correctly if they appeared on the list but bug out if there's nothing, and it never navigates to Set Filters.
I noticed that it clicks on the bot icon in my taskbar and even when I reorder my taskbar it will still click on it for some reason.


https://github.com/christianciu/gbf-automation-updated/assets/20892295/b7767902-4966-4e1b-b38c-4f0bb31e16b9

**Comment by christianciu** [comment]
@ChikyuTenshi 

1. I can't see part of the video, its just get blured
![image](https://github.com/christianciu/gbf-automation-updated/assets/53420284/bca3c07a-df8f-44fb-bf59-ff4402bc3687)

2. About raid list filters, its intentional as its buggy on the original version and failed to choose the right raid (https://github.com/steve1316/granblue-automation-pyautogui/pull/207). Filter the raid you want first then start the bot (it also ignore your choices of farmed items & raid battle)
3. As for clicking bot in taskbar ..... i have no idea ? It never happened to me and i found its strange that the logs say it found a raid while the raid filter still unset yet

**Comment by ChikyuTenshi** [comment]
@christianciu 

1. I blurred it on purpose because it was showing my name + crew but it was just scrolling down as it should, Not sure why it can't locate the banner.
2. Okay I thought so too.
3. Alright so I found a temporary workaround for now, just adding a bunch of icons to my taskbar to make sure the bot isn't directly below the game makes it hit reload properly. Maybe lowering the scan range will help?

**Comment by christianciu** [comment]
> @christianciu
> 
> 1. I blurred it on purpose because it was showing my name + crew but it was just scrolling down as it should, Not sure why it can't locate the banner.
> 2. Okay I thought so too.
> 3. Alright so I found a temporary workaround for now, just adding a bunch of icons to my taskbar to make sure the bot isn't directly below the game makes it hit reload properly. Maybe lowering the scan range will help?

1. Its working for me. Are you sure your Arcanum banner is in the home page ?
3. Gotta be honest with you, i don't really understand the original code for OCR & scanning and havent got time (and motivation) for learning it (Im seasonal so thats that). If you want, you could try lowering it. No need to install tauri dependencies if you are only working on python files. Just replace the old file with new ones and run the app

**Comment by ChikyuTenshi** [comment]
1. Yes it is there, I even verified how Arcarum_Banner looks and it's the same one at my homepage
![Screenshot 2024-05-09 220036](https://github.com/christianciu/gbf-automation-updated/assets/20892295/422b8866-a905-4f4f-8b12-cc8c5c71f40d)
2. Understandable, I'll try tinkering with it and see how it goes

**Comment by christianciu** [comment]
> 1. Yes it is there, I even verified how Arcarum_Banner looks and it's the same one at my homepage
>    ![Screenshot 2024-05-09 220036](https://private-user-images.githubusercontent.com/20892295/329357428-422b8866-a905-4f4f-8b12-cc8c5c71f40d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUzMDEwOTAsIm5iZiI6MTcxNTMwMDc5MCwicGF0aCI6Ii8yMDg5MjI5NS8zMjkzNTc0MjgtNDIyYjg4NjYtYTkwNS00ZjRmLThiMTItY2M4YzVjNzFmNDBkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEwVDAwMjYzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk5ZjAzOTBiZWY0YTdiZWE1NWZlMTU3ZjE4Mjg4NWQ5NGYxMWNlMTE5MjYzOWNmYTIyZmQyN2Y5ZGM0MWVmZGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Nfty3QhXD1skcP-vjbRaAgs1jUsJA-8D-qNulFk0Ty0)
> 2. Understandable, I'll try tinkering with it and see how it goes

If you confirm the banner is there, but they can't find it. You could screenshot the banner and then replace the banner in the folder with your screenshotted one.  Its the same thing i do with adding new summons.

**Comment by ChikyuTenshi** [comment]
Yeah after posting my last comment I tried doing so and it works flawlessly
